### PR TITLE
Mark more texts for translation

### PR DESCRIPTION
**Pull request by lslezak**
Just marking more texts for translation

### CODE REVIEW

File: web/src/components/software/ProductSelectionPage.jsx [hunk 1]
```diff
@@ -88,16 +90,27 @@ function ProductSelectionPage() {
...
         </Button>
       </MainActions>
 
       <Form id="product-selector" onSubmit={accept}>
-        <FormGroup isStack label={`Choose a product (${products.length} available)`} role="radiogroup">
+        <FormGroup
+          isStack
+          label={
+            format(
+              n_("Choose a product (%d available)", "Choose a product (%d available)", products.length),
```
Comment (lslezak) [inline web/src/components/software/ProductSelectionPage.jsx:108]: Here I'm not sure if we really need to display the number of available products.

- There are usually just few products
- The number does not provide any meaningful information for the user, you can always choose *only one* product, it does not matter how many products are available

So I'd rather make it easier and use just simple `_("Choose a product")` here.

Opinions?

Comment (imobachgs) [reply web/src/components/software/ProductSelectionPage.jsx:108]: Yes, I agree, I would get rid of the number of available products.

**Review by imobachgs — APPROVED** [review]

File: web/src/components/software/ProductSelectionPage.jsx [hunk 2]
```diff
@@ -88,16 +89,18 @@ function ProductSelectionPage() {
 
   return (
     <>
-      <Title>Product selection</Title>
+      {/* TRANSLATORS: page header */}
+      <Title>{_("Product selection")}</Title>
```
Comment (jreidinger) [inline web/src/components/software/ProductSelectionPage.jsx:93]: not sure if in title it should not be all capitalized like `Product Selection`

Comment (imobachgs) [reply web/src/components/software/ProductSelectionPage.jsx:93]: Why? Do we have any guidelines about that?

**Pull request merged**